module/VuFind/src/VuFind/ILS/Driver/Demo.php

```diff
@@ -893,6 +893,9 @@ public function patronLogin($username, $password)
     public function getMyProfile($patron)
     {
         $this->checkIntermittentFailure();
+        $age = 13 + rand(0, 100);
```

Comment:
Silly observation, but why do math here? Could just be `rand(13, 113)` (or maybe the more realistic `rand(13, 100)`).

Reply:
That was to make it clear that 13 is the minimum age a Demo user can have, but admittedly your proposal is not that much harder to read, so I'll just change it.

Reply:
This changed as well.